packaging/Containerfile.fedora

```diff
@@ -0,0 +1,19 @@
+FROM quay.io/centos-bootc/fedora-bootc:eln
+
+COPY rpmbuild/RPMS/x86_64/flightctl-agent-0.0.1-1.el9.x86_64.rpm /tmp/
+
+COPY packaging/flightctl-public-service/flightctl_rsa.pub /usr/etc-system/root.keys
+RUN touch /etc/ssh/sshd_config.d/30-auth-system.conf; \
+    mkdir -p /usr/etc-system/; \
+    echo 'AuthorizedKeysFile /usr/etc-system/%u.keys' >> /etc/ssh/sshd_config.d/30-auth-system.conf; \
+    chmod 0600 /usr/etc-system/root.keys
+VOLUME /var/roothome
+
+ADD packaging/flightctl-public-service/config.yaml /etc/flightctl/
+ADD packaging/flightctl-public-service/ca.crt /etc/flightctl
+ADD packaging/flightctl-public-service/client-enrollment.* /etc/flightctl/
+
+RUN rpm-ostree install -y /tmp/flightctl-agent-0.0.1-1.el9.x86_64.rpm
+RUN ln -s /usr/lib/systemd/system/podman.socket /usr/lib/systemd/system/multi-user.target.wants/
+RUN ln -s /usr/lib/systemd/system/flightctl-agent.service /usr/lib/systemd/system/multi-user.target.wants/
```

Comment:
Shouldn't these symlinks be created from the RPM spec's `%install` section?
Also, are we missing a symlink from the bannerfile in `/run/flightctl-banner` into `%{buildroot}%{_sysconfdir}/issue.d/flightctl.issue`?

Reply:
Having flightctl-agent service enabled by default (via rpm) is a decision to be made, of course. If we all agree that's the way to go, I can change it. No problem!

Regarding the banner, #33 just took care of that, by creating the QR code in there.

Reply:
If we write the file directly to `/etc` now, why still keep the `/run` one?

Reply:
right, I don't know why @mangelajo chose that location. If it was an arbitrary one, we can certainly remove it.

Reply:
Probably because `transmission` used a symlink, which it did because `zezere` did it back then. I don't recall the reason anymore, really. If there was a good reason, I could imagine it was around SELinux (re)labeling or simply that /etc is for config and processes shouldn't be writing to it really.

Reply:
I don't know :D

Reply:
Ok, having the agent write it directly, the SELinux context becomes `unconfined_u:object_r:etc_t:s0` instead of `system_u:object_r:etc_t:s0`, but that should be fine. Let's remove the code that writes the duplicate banner to `/run` then?

Reply:
Ok, I'll add that in a separate PR to avoid mixing fixes and have a better tracking history. Is that fine?

Reply:
#38